gallery/src/components/NxGlobalSidebarFooter/NxGlobalSidebarFooterPage.tsx

```diff
@@ -0,0 +1,138 @@
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import CodeExample from '../../CodeExample';
+import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
+import { NxP, NxCode, NxTable, NxTile, NxH2, NxH3 } from '@sonatype/react-shared-components';
+
+const NxGlobalSidebarFooterExample = require('./NxGlobalSidebarFooterExample.tsx?raw'),
+    NxGlobalSidebarFooterScrollingExample = require('./NxGlobalSidebarFooterScrollingExample.tsx?raw'),
+    NxGlobalSidebarFooterMinimalExample = require('./NxGlobalSidebarFooterMinimalExample.tsx?raw');
+
+export default function NxGlobalSidebarPage() {
+  return (
+    <>
+      <GalleryDescriptionTile>
+        <NxP>
+          <NxCode>NxGlobalSidebarFooter</NxCode> is a component intended to be used in conjunction with
+          {' '}<NxCode>NxGlobalSidebar</NxCode>. It appears at the bottom of the sidebar and displays meta-information
+          about the application such as links to documentation, versions, and branding.
+        </NxP>
+        <NxTile.Subsection>
+          <NxTile.SubsectionHeader>
+            <NxH3><NxCode>NxGlobalSidebarFooter</NxCode> Props</NxH3>
+          </NxTile.SubsectionHeader>
+          <NxTable>
+            <NxTable.Head>
+              <NxTable.Row>
+                <NxTable.Cell>Prop</NxTable.Cell>
+                <NxTable.Cell>Type</NxTable.Cell>
+                <NxTable.Cell>Required</NxTable.Cell>
+                <NxTable.Cell>Details</NxTable.Cell>
+              </NxTable.Row>
+            </NxTable.Head>
+            <NxTable.Body>
+              <NxTable.Row>
+                <NxTable.Cell>supportText</NxTable.Cell>
+                <NxTable.Cell>string</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>
+                  Alternate text that should appear in the support link, by default the text reads "Help and Support".
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>supportLink</NxTable.Cell>
+                <NxTable.Cell>string</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>
+                  URL that points to the help or support documents for this product. This prop is required for the
+                  support text to appear.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>releaseText</NxTable.Cell>
+                <NxTable.Cell>String</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>
+                  Text that indicates what version of the software is currently being used. You may use just a version
+                  number or a mix of text and a number, e.g. "Release 3.1.4".
```

Comment:
This is where I think we should mention that this would most typically be the product/application name followed by the version.

Reply:
> This is where I think we should mention that this would most typically be the product/application name followed by the version.

I'm not sure that's true though, I think we should leave it open. It could be the product name, it could be something like "version", "release", or even "v".

Reply:
> I'm not sure that's true though, I think we should leave it open. It could be the product name, it could be something like "version", "release", or even "v".

That's why I said "most typically"

Reply:
> > I'm not sure that's true though, I think we should leave it open. It could be the product name, it could be something like "version", "release", or even "v".
> 
> That's why I said "most typically"

I'm not even sure that "most typically" is true, we won't know until we see what the designers design. I have expanded the documentation though and used the "most typically" language: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/fc8acd2c5de712af6bb1e38e480c7f64f901a793